tests/page-objects/admin/languages.wdio.page.js

```diff
@@ -0,0 +1,62 @@
+const utils = require('../../utils');
+
+const submitButton = () => $('div.form-actions button.btn.btn-primary');
+const addLanguageButton = () => $('span=Add new language');
+const addLanguageModal = () => $('body.ng-scope.modal-open');
+const languageCodeInput = () => $('[ng-model="language.code"]');
+const languageNameInput  = () => $('[ng-model="language.name"]');
+const languageSubmitButton  = () => $('a.btn.submit.ng-scope.ng-binding.btn-primary');
+const applicationLink = () => $('span=Application');
+const defaultLanguageDropdown = () => $('#locale');
+const outgoingLanguageDropdown = () => $('#locale-outgoing');
+
+const goToLanguagesTab = async () => {
+  await browser.url(utils.getAdminBaseUrl() + 'display/languages');
+  await browser.refresh();
+  await (await addLanguageButton()).waitForDisplayed();
+};
+
+const openAddLanguageModal = async () => {
+  await (await addLanguageButton()).click();
+  await (await addLanguageModal()).waitForDisplayed();
+};
+
+const addNewLanguage = async (code, name) =>{
+  await (await addLanguageButton()).click();
+  await (await addLanguageModal()).waitForDisplayed();
+  await (await languageCodeInput()).waitForDisplayed();
+  await (await languageCodeInput()).setValue(code);
+  await (await languageNameInput()).setValue(name);
+  await (await languageSubmitButton()).click();
+  await (await languageSubmitButton()).waitForDisplayed({reverse:true});
+};
+
+const languageDisplayed = async (code) =>{
+  const languageDiv = () => $(`#locale-${code}`);
+  const languageName = await (await languageDiv()).getText();
+  return languageName;
+};
+
+const selectLanguage = async (element, code) => {
+  await (await element()).selectByAttribute('value', `string:${code}`);
+  await (await submitButton()).click();
+  await browser.waitUntil(async () => (await $('.success.ng-binding.ng-scope').getText()) === 'Saved');
```

Comment:
Curiously, this fails for me locally with: 
```
Error: waitUntil condition failed with the following reason: $(...).getText is not a function
    at /home/diana/projects/medic-second/node_modules/webdriverio/build/commands/browser/waitUntil.js:70:15
    at async Browser.wrapCommandFn (/home/diana/projects/medic-second/node_modules/@wdio/utils/build/shim.js:78:29)
    at async Object.selectLanguage (/home/diana/projects/medic-second/tests/page-objects/admin/languages.wdio.page.js:43:3)
    at async Context.<anonymous> (/home/diana/projects/medic-second/tests/e2e/translations/new-language.wdio-spec.js:37:12)
    at async Context.executeAsync (/home/diana/projects/medic-second/node_modules/@wdio/utils/build/shim.js:136:16)
    at async Context.testFrameworkFnWrapper (/home/diana/projects/medic-second/node_modules/@wdio/utils/build/test-framework/testFnWrapper.js:52:18)
````
I think this should be: 
```suggestion
  await browser.waitUntil(async () => await (await $('.success.ng-binding.ng-scope')).getText() === 'Saved');
  ```
  

```